### ISSUE TITLE
Moby/Docker relationship link is broken

### ISSUE THREAD

**Issue by thanatos** [moby/moby#34291]
There is a giant link at the top of the README:

> Docker users, see [Moby and Docker](https://mobyproject.org/#moby-and-docker) to clarify the relationship between the projects

The link has a target, `#moby-and-docker`, but that seems to be broken. The landing page doesn't *really* clarify what the differences are, perhaps Moby is some base library to Docker? (But then what is `libcontainerd`? More abstraction?)

**Comment by AkihiroSuda** [comment]
cc @moby/mobywebsite

**Comment by vdemeester** [comment]
cc @chanezon

**Comment by mataneja** [comment]
@thanatos: 
The link is working fine for me in Chrome, Safari and Firefox. Is your concern that the link is not working or the new page does not explicitly call out the relationship between Moby and Docker?

**Comment by thaJeztah** [comment]
Yes the `#moby-and-docker` anchor is no longer on that page (previously there was a section describing the relation between Docker and the Moby project)
